docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,235 @@
+
+3) **Own computing environment** - PC, local cluster. Data needs to be downloaded and all tools must be installed to this system. On the other hand, it gives more freedom to select the tools and set-up. Usually this does not cause any extra costs, but the computing power is usually rather limited.
+
+CSC services do not fit well in this categorization, as they provide some features from all of these. **CSC computing services provide a lot of computing power and storage space, and they are free of charge** for Finnish researchers for academic or educational use. 
+
+At CSC, EO data can be processed and analyzed using a supercomputer, for example [supercomputer Puhti](../../../computing/systems-puhti.md), or a virtual machine in the [cPouta cloud service](../../../cloud/pouta/pouta-what-is.md). Puhti's computing capacity can hardly be compared to any other EO service, in both available processing power and amount of memory. Both Puhti and cPouta have also GPU resources, which are especially useful for large simulations and deep learning use cases. 
+
+Puhti has also a lot of [pre-installed applications](#what-applications-are-available-on-puhti), so it is an environment ready to use. cPouta virtual machines are similar to commercial cloud services, where all set-up and installations are done by the end-user. In general, both services only support Linux software.
+
+At CSC, [some Finnish EO datasets](#eo-data-at-csc) are available for direct use. In many cases, however, downloading EO data from other services (see [list of EO data download services](#eo-data-download-services)) is a required step of the process. Puhti and cPouta provide local storage of ~1-20 Tb. For more storage space,  [Allas object storage](../../../data/Allas/index.md) can be used.
+
+Using CSC computing services requires basic Linux skills and ability to use some scripting language or command-line tools. In addition, supercomputers and virtual machines require you to understand some specific concepts, so it takes a few hours to get started. The new Puhti web interface makes the start considerably easier, providing a desktop environment in the web browser, which enables the use of tools with Graphical User Interfaces (GUI) and also tools like R Studio and JupyterLab for an easy start with R, Python and Julia.
+
+## What data do I need?
+
+What to consider when chosing data:
+
+* Sensor
+    * Optical
+    * Multispectal
+    * Hyperspectral
+    * RADAR (Radio Detection and Ranging), SAR (Synthetic Aperture Radar)
+    * LiDAR (Light Detection and Ranging)
+* Resolution
+    * Temporal: when and how often a certain area is visited
+    * Spatial: the area on the ground that each pixel covers
+    * Spectral: the area of the electromagnetic spectrum that is observed and spectral width of each band provided
+        * Depending on the wavelengths observed,  clouds and atmospheric artifacts may result in data gaps
```

Comment:
Move this under 'Sensor', since it applies also to RADAR: _* Depending on the wavelengths observed, clouds, ground conditions and atmospheric artifacts may result in data gaps_